numpy/ma/tests/test_core.py

```diff
@@ -447,6 +447,13 @@ def test_deepcopy(self):
         assert_equal(copied.mask, [0, 0, 0])
         assert_equal(a.mask, [0, 1, 0])
 
+    def test_fstring(self):
+        # fstrings (PEP 498) are a SyntaxError in versions under Py 3.6.
+        # Seeing that as of 20200123, the atravis do not target versions under 3.6, so this is ok.
+        a = array([0, 1, 2], mask=[False, True, False])
+        assert_equal(f"{a}", "[0 -- 2]")
+        assert_equal(f"{masked}", "--")
```

Comment:
Even though they are supported, I think we should be testing this at a more primitive level:
```suggestion
        assert_equal(format(masked), "--")
```